mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTest.java

```diff
@@ -134,7 +138,17 @@ protected void initializeMailboxManager() throws Exception {
             new SimpleGroupMembershipResolver(),
             new MessageParser(),
             messageIdFactory);
-        messageIdManager = new InMemoryMessageIdManager(storeMailboxManager);
+        DefaultDelegatingMailboxListener delegatingListener = new DefaultDelegatingMailboxListener();
+        MailboxEventDispatcher mailboxEventDispatcher = new MailboxEventDispatcher(new DefaultDelegatingMailboxListener());
```

Comment:
You should use `delegatingListener` instead of building a new one

Reply:
ok